pkg/config/serving/server.go

```diff
@@ -58,6 +59,7 @@ func ToServerConfig(ctx context.Context, servingInfo configv1.HTTPServingInfo, a
 	if !authorizationConfig.Disabled {
 		authorizationOptions := genericapiserveroptions.NewDelegatingAuthorizationOptions()
 		authorizationOptions.RemoteKubeConfigFile = kubeConfigFile
+		authorizationOptions.AllowCacheTTL = authorizationConfig.AllowCacheTTL.Duration
```

Comment:
what is the default? None?

Reply:
it's 10s as set by the `NewDelegatingAuthenticationOptions`/`NewDelegatingAuthorizationOptions` functions